onnxruntime/core/session/inference_session.h

```diff
@@ -202,7 +116,9 @@ class InferenceSession {
     */
   common::Status Run(const NameMLValMap& feeds,
                      const std::vector<std::string>& output_names,
-                     std::vector<MLValue>* p_fetches);
+                     std::vector<MLValue>* p_fetches) override {
+    return Run({}, feeds, output_names, p_fetches);
```

Comment:
>{} [](start = 15, length = 2)

I'd call RunOptions type explicitly so it is never confused with a init_list for vector or otherwise.

Reply:
This is exquisite. I'll try it in next iteration, thanks.